File: Programmers/LEVEL_1/문자열 내 p와 y의 개수.cpp
#include <string>
#include <iostream>
using namespace std;

bool solution(string s)
{
    bool answer = true;
    int p_size = 0;
    int y_size = 0;

    for (int i = 0; i < s.size(); i++)
    {
        if (s[i] == 'p' || s[i] == 'P')
            p_size++;
        if (s[i] == 'y' || s[i] == 'Y')
            y_size++;
    }
    if (p_size != y_size) answer = false;

    return answer;
}

File: Programmers/LEVEL_1/문자열 내 p와 y의 개수.js
function solution(s) {
  var answer = true;

  let count1 = 0;
  let count2 = 0;

  for (let i of s) {
    if (i == "p" || i == "P") count1++;
    if (i == "y" || i == "Y") count2++;
  }

  if (count1 != count2) answer = false;

  return answer;
}
